webapp/src/modules/mortgage/utils.js

```diff
@@ -21,11 +30,39 @@ export function getLoanMetadata() {
   return `#mortgage #required-cosigner:${mortgageManagerContract.address}`
 }
 
-export function getActiveMortgagesByBorrower(mortgages, borrower) {
-  return mortgages.filter(
-    mortgage =>
-      mortgage &&
-      mortgage.borrower === borrower &&
-      mortgage.status !== MORTGAGE_STATUS.cancelled
+export function isMortgageActive(mortgage, parcel) {
+  return (
+    mortgage &&
+    parcel &&
+    ((isMortgagePending(mortgage) &&
+      isOpen(parcel.publication, PUBLICATION_STATUS.open)) ||
+      hasStatus(mortgage, [MORTGAGE_STATUS.ongoing, MORTGAGE_STATUS.paid]))
```

Comment:
I think this is kind of confusing I can't see what it's doing at first glance. Can we maybe reword it?

Like

```javascript
  if (mortgage && parcel) {
    const isPending = isMortgagePending(mortgage) && isOpen(parcel.publication, PUBLICATION_STATUS.open)
    
    return isPending || hasStatus(mortgage, [MORTGAGE_STATUS.ongoing, MORTGAGE_STATUS.paid])
  }
```

(wrote on Github, don't take seriously)